tests/Datagrid/PagerTest.php

```diff
@@ -54,50 +52,4 @@ public function testComputeNbResultForCompositeId(string $className): void
         $pager->setQuery($pq);
         $this->assertSame(0, $pager->computeNbResult());
     }
-
-    public function dataGetComputeNbResult(): array
-    {
-        return [
-            [true],
-            [false],
-        ];
-    }
-
-    /**
-     * @dataProvider dataGetComputeNbResult
-     */
-    public function testComputeNbResult(bool $distinct): void
-    {
-        $query = $this->getMockBuilder(AbstractQuery::class)
-            ->disableOriginalConstructor()
-            ->setMethods(['getResult'])
-            ->getMockForAbstractClass();
-
-        $query->expects($this->once())
-            ->method('getResult')
-            ->willReturn([['cnt' => 1], ['cnt' => 2]]);
-
-        $queryBuilder = $this->getMockBuilder(QueryBuilder::class)
-            ->disableOriginalConstructor()
-            ->setMethods(['getQuery', 'select', 'resetDQLPart'])
-            ->getMock();
-
-        $queryBuilder->expects($this->once())
-            ->method('getQuery')
-            ->willReturn($query);
-
-        $queryBuilder->expects($this->once())
-            ->method('select');
-
-        $proxyQuery = new ProxyQuery($queryBuilder);
-        $proxyQuery->setDistinct($distinct);
-
-        $queryBuilder->expects($this->once())
-            ->method('resetDQLPart')
-            ->willReturn($proxyQuery);
-
-        $pager = new Pager();
-        $pager->setQuery($proxyQuery);
-        $pager->computeNbResult();
-    }
```

Comment:
Why is all this removed?

Reply:
- This test is using a `Sonata\DoctrineORMAdminBundle\Tests\Filter\QueryBuilder` instead of a Doctrine one.
- This test, which seems complicated, is not really testing something (or something useful), there are some `expects` about the implementation, nothing about the result of this method.
- There is already another test on the `computeNbResult` which is doing better assertions.

Reply:
Please mention this in the body of your commit message.

Reply:
Done @greg0ire